File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/L. GCD/main.dart
import 'dart:io';
import 'dart:math';

void main() {
  List<String> input = stdin.readLineSync()!.split(" ");
  int a = int.parse(input[0]);
  int b = int.parse(input[1]);
  int mn = min(a, b);
  int gcd = 1;
  for (int i = 1; i <= mn; i++) {
    if (a % i == 0 && b % i == 0) gcd = i;
  }
  print(gcd);
}

File: Codeforces/1. ICPC Assiut University Community/Sheet #2 (Loops)/L. GCD/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/L

using System;
using static System.Console;
using static System.Math;

namespace Codeforces.Assuit.Contest1
{
    class ProblemL
    {
        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(" ");
            ushort a = ushort.Parse(input[0]);
            ushort b = ushort.Parse(input[1]);
            ushort min = Min(a, b);
            ushort gcd = 1;
            for (ushort i=1; i <= min; i++){
                if(a%i==0 && b%i==0) gcd = i;
            }
            WriteLine(gcd);
        }
    }
}
